messageTransformer/messageTransformer.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
```

Comment:
nit: 2023

Reply:
oh, that means I probably need to bump something else.